### PR TITLE
[AI Partners Ecosystem] - Lettria integration

### PR DESCRIPTION
**Pull request by eleapttn**
Create a new section in AI & Machine Learning doc:
**AI Partners Ecosystem**

Add 2 guides about Lettria partner:
- `ecosystem_01_lettria_billing_features_capabilities` to explain how Lettria services work with AI Deploy, how it is billed and what are the capabilities and limitations
- `ecosystem_02_lettria_models` to explain the partner AI models in detail (provided by partner)

FYI:
- 🇬🇧 only the `guide.en-gb.md` have been created
- 🔗 `index.md` is updated with the new guides and section
- ✅ `meta.yaml` files created

⚠️ Is it possible to deliver it on Monday 10 July please? (prod start date for us)

### CODE REVIEW

File: pages/platform/ai/ecosystem_01_lettria_billing_features_capabilities/guide.en-gb.md [hunk 1]
```diff
@@ -0,0 +1,315 @@
...
+
+```console
+Resources price = (nb app) x (nb resources) x (nb replicas) x [ (unit price per hour HT) x (nb hour) ]
+```
+
+To learn more about the basic cost of an app launched with AI Deploy, please refer to this [documentation](/pages/platform/ai/deploy_guide_06_billing_concept).
+
+#### Lettria model licenses price
+
+The second step consists in calculating the cost of one of the Lettria partner service according to the chosen resource.
```
Comment (OVH-Team-Guides) [inline pages/platform/ai/ecosystem_01_lettria_billing_features_capabilities/guide.en-gb.md]: I think at least one of the nouns here should be in plural.
```suggestion
The second step consists in calculating the cost of one of the Lettria partners services according to the chosen resource.
```

File: pages/platform/ai/ecosystem_01_lettria_billing_features_capabilities/guide.en-gb.md [hunk 2]
```diff
@@ -0,0 +1,315 @@
...
+> [!primary]
+>
+> To learn more about the basic cost (resource price) of an app launched with AI Deploy, please refer to this [documentation](/pages/platform/ai/deploy_guide_06_billing_concept).
+>
+> Please refer to the [OVHcloud Public Cloud website](https://www.ovhcloud.com/en-gb/public-cloud/prices/) for all information about resources and partners models prices.
+>
+
+### Examples
+
+#### Example 1: 1 GPU app with for 10 hours then deleted
```
Comment (OVH-Team-Guides) [inline pages/platform/ai/ecosystem_01_lettria_billing_features_capabilities/guide.en-gb.md]: Looks like there is either one extra word or one missing word

File: pages/platform/ai/ecosystem_02_lettria_models/guide.en-gb.md [hunk 3]
```diff
@@ -0,0 +1,346 @@
...
+
+The Sentiment Analysis API facilitates sentiment assessment in diverse content, including:
+
+- customer feedback
+- social media posts
+- product reviews
+
+By attributing a numerical value to the sentiment of text, it provides an understanding of the overall positivity or negativity of the content.
+
+The API enables objective measurement of sentiment in large data sets, aiding informed decision-making and timely actions. It caters to needs such as brand reputation monitoring, market research and customer experience optimization, empowering you with actionable insights.
```
Comment (OVH-Team-Guides) [inline pages/platform/ai/ecosystem_02_lettria_models/guide.en-gb.md]:
```suggestion
The API enables objective measurement of sentiment in large data sets, helping informed decision-making and timely actions. It caters to needs such as brand reputation monitoring, market research and customer experience optimization, empowering you with actionable insights.
```

File: pages/platform/ai/ecosystem_02_lettria_models/guide.en-gb.md [hunk 4]
```diff
@@ -0,0 +1,346 @@
...
+
+Our API delivers sentiment values from -1 (signifying strong negative sentiment) to 1 (indicating strong positive sentiment), facilitating comprehensive understanding of sentiment intensity. This allows you to identify sentiment polarity and measure its degree in the text.
+
+The API's user-friendly design allows effortless integration into your applications, unlocking the capabilities of sentiment analysis. Enhance customer satisfaction, take data-driven decisions and optimize business strategies with sentiment analysis. Discover and understand the true sentiment behind text, revolutionizing your audience engagement.
```
Comment (OVH-Team-Guides) [inline pages/platform/ai/ecosystem_02_lettria_models/guide.en-gb.md:42]: The 2nd and 3rd sentences "sound" very marketing-oriented, maybe too much for documentation purposes.

File: pages/platform/ai/ecosystem_01_lettria_billing_features_capabilities/guide.en-gb.md (hunk 2 above)
Comment (eleapttn) [inline pages/platform/ai/ecosystem_01_lettria_billing_features_capabilities/guide.en-gb.md]: #### Example 1: 1 GPU app for 10 hours then deleted

Comment (eleapttn) [inline pages/platform/ai/ecosystem_01_lettria_billing_features_capabilities/guide.en-gb.md]:
```suggestion
#### Example 1: 1 GPU app for 10 hours then deleted
```

File: pages/platform/ai/ecosystem_02_lettria_models/guide.en-gb.md (hunk 4 above)
Comment (eleapttn) [reply pages/platform/ai/ecosystem_02_lettria_models/guide.en-gb.md:42]: Indeed, I agree with you... But it's the partner's doc and we didn't write that part of the guide

**Pull request merged**